views/error_handlers/404.ejs

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head> 
+    <% const title = 'Page Not found' %> 
+    <%- include('../partials/head', {title}); %>
```

Comment:
There's probably no need to store the title here in a const:
```
<%- include('../partials/head', {title: 'Page Not found'}); %>
```

Reply:
Done.